bridge/client/app/app.module.ts

```diff
@@ -94,6 +94,7 @@ import { KtbSubscriptionComponent } from './_components/ktb-subscription/ktb-sub
 import { DtFilterFieldModule } from '@dynatrace/barista-components/filter-field';
 import { KtbSubscriptionItemComponent } from './_components/ktb-subscription-item/ktb-subscription-item.component';
 import { KtbDeploymentListComponent } from './_components/ktb-deployment-list/ktb-deployment-list.component';
+import {KtbUserComponent} from './_components/ktb-user/ktb-user.component';
```

Comment:
Please align the formatting with the other imports in this file (I know, we're missing linting in our CI...)

Reply:
@christian-kreuzberger-dtx  solved through the latest commit